packages/fauna/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@next-auth/fauna-adapter",
+  "version": "0.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

Comment:
Could you add these to package.json:

```json
"private": false,
"publishConfig": {
  "access": "public"
},
```
It seems t be required to be able to publish scoped packages.